PathPlanning/ModelPredictiveTrajectoryGenerator/trajectory_generator.py

```diff
@@ -18,7 +18,7 @@
 
 # optimization parameter
 max_iter = 100
-h = np.array([0.5, 0.02, 0.02]).T  # parameter sampling distance
+h: np.ndarray = np.array([0.5, 0.02, 0.02]).T  # parameter sampling distance
```

Comment:
Why this type annotation is needed?

Reply:
test_mypy_type_check.py fails on removing the annotation. That was the reason CI pipeline failed.